downtimes.go

```diff
@@ -0,0 +1,210 @@
+package mackerel
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+// Downtime information
+type Downtime struct {
+	ID                   string              `json:"id,omitempty"`
+	Name                 string              `json:"name,omitempty"`
```

Comment:
Hmm, imo those mandatory parameters should not be tagged as omitempty. How do you think?

Reply:
True. I updated the json tags based on the api docs (for find out required fields).